k8s.io/README.md

```diff
@@ -57,23 +57,18 @@ Redirections
 
 NOTE: please see configmap-nginx.yaml for rewrite rules.
 
-Testing
+How to deploy
 ====
-Configure kubectl to target a test cluster on GKE.
 
-Run `make deploy-fake-secret deploy` and wait for the service to be available--
-the load balancer may take some time to configure.
+1) Log into Google Cloud Shell.  Our clusters do not allow access from the
+   internet.
 
-Set `TARGET_IP` to the ingress IP of the running service:
+2) Configure `kubectl` to have a cluster named "prod-aaa".  Running
```

Comment:
@thockin using the console the `connect` button tells me that i should use:
```gcloud container clusters get-credentials aaa --region us-central1 --project kubernetes-public```

When i do that, the name is `gke_kubernetes-public_us-central1_aaa`

So then i'd need to do `kubectl --cluster=gke_kubernetes-public_us-central1_aaa get ns` 

what's the best way to get the name to be what the script expects - namely `prod-aaa`?

Reply:
Good point.  I did that manually.  Will make it easier.  Take a look now